src/assets/scripts/client/game/GameOptions.js

```diff
@@ -63,15 +67,21 @@ export default class GameOptions {
     }
 
     /**
+     * Sets a game option to a given value
+     *
      * @for GameOptions
      * @method set
-     * @param name {string}
-     * @param value
+     * @param name {string} name of the option to change
+     * @param value {string} value to set the option to
      */
     set(name, value) {
         localStorage[`zlsa.atc.option.${name}`] = value;
         this[name] = value;
 
+        if (this._options[name].onChangeEvent) {
```

Comment:
discussed, and will leave in place with new name, and `_noop` when no operation is needed.

done